cmd/stop.go

```diff
@@ -157,6 +157,10 @@ func runStopSuperNodeSubCommand(ctx context.Context, config *configs.Config) {
 
 	// *************  Kill process dd-service  *************
 	stopDDService(ctx)
+	//stopService(ctx, constants.DDService)
```

Comment:
should change to stopService(ctx, constants.DDService)?

Reply:
It is related to the python module service of `dd-service`, thus `stopDDService` needs to be called instead.